bindata/v3.11.0/openshift-controller-manager/ingress-to-route-controller-clusterrole.yaml

```diff
@@ -12,6 +12,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
hm, this will make it possible to reach leases in all namespaces since we are creating cluster role binding here https://github.com/openshift/cluster-openshift-controller-manager-operator/blob/master/bindata/v3.11.0/openshift-controller-manager/ingress-to-route-controller-clusterrolebinding.yaml . Not sure this is what we want